test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/RazorDirectivesTest/DirectiveDescriptor_NoErrorsSemicolonAfterDirective.stree.txt

```diff
@@ -1,20 +1,24 @@
-Markup block - Gen<None> - 19 - (0:0,0)
-    Markup span - Gen<Markup> - [] - SpanEditHandler;Accepts:Any - (0:0,0) - Tokens:1
-        SyntaxKind.Unknown;[];
-    Directive block - Gen<Directive:{custom;SingleLine;Unrestricted}> - 19 - (0:0,0)
-        Transition span - Gen<None> - [@] - SpanEditHandler;Accepts:None - (0:0,0) - Tokens:1
-            SyntaxKind.Transition;[@];
-        MetaCode span - Gen<None> - [custom] - SpanEditHandler;Accepts:None - (1:0,1) - Tokens:1
-            SyntaxKind.Identifier;[custom];
-        Markup span - Gen<None> - [ ] - SpanEditHandler;Accepts:Whitespace - (7:0,7) - Tokens:1
-            SyntaxKind.Whitespace;[ ];
-        Code span - Gen<DirectiveToken {;String;Opt:False}> - ["hello"] - DirectiveTokenEditHandler;Accepts:NonWhitespace - (8:0,8) - Tokens:1
-            SyntaxKind.StringLiteral;["hello"];
-        None span - Gen<None> - [ ] - SpanEditHandler;Accepts:Whitespace - (15:0,15) - Tokens:1
-            SyntaxKind.Whitespace;[ ];
-        MetaCode span - Gen<None> - [;] - SpanEditHandler;Accepts:Whitespace - (16:0,16) - Tokens:1
-            SyntaxKind.Semicolon;[;];
-        Markup span - Gen<None> - [  ] - SpanEditHandler;Accepts:Whitespace - (17:0,17) - Tokens:1
-            SyntaxKind.Whitespace;[  ];
-    Markup span - Gen<Markup> - [] - SpanEditHandler;Accepts:Any - (19:0,19) - Tokens:1
-        SyntaxKind.Unknown;[];
+RazorDocument - [0..19)::19 - [@custom "hello" ;  ]
+    MarkupBlock - [0..19)::19
+        MarkupTextLiteral - [0..0)::0 - [] - Gen<Markup> - SpanEditHandler;Accepts:Any
+            Marker;[];
+        CSharpCodeBlock - [0..19)::19
```

Comment:
Still hate that we wrap things like this in code blocks but it's something we'll probably never change